### ISSUE TITLE
image use zn name in md build faild

### ISSUE THREAD

**Issue by wefesenlin** [vuejs/vitepress#1021]
### Describe the bug

references image with zn name in .md  like this  and build docs

<img width="856" alt="image" src="https://user-images.githubusercontent.com/9570544/179921950-89313452-7a6c-4fd4-8d19-fdb81ab5b273.png">

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/9570544/179922042-d8b31c41-ad2b-4ae2-8923-9d988ab5dd6f.png">

### Reproduction

1、references image with zn name in .md 
2、build docs

### Expected behavior

build success

### System Info

```shell
System:
    OS: macOS 11.6
    CPU: (8) x64 Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz
    Memory: 65.65 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 15.14.0 - ~/.nvm/versions/node/v15.14.0/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 7.24.0 - ~/.nvm/versions/node/v15.14.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Browsers:
    Chrome: 103.0.5060.114
    Safari: 14.1.2
  npmPackages:
    vitepress: ^1.0.0-alpha.4 => 1.0.0-alpha.4
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by brc-dd** [comment]
Hi! It was fixed by #933. The fix will be there in the next version.

**Comment by wefesenlin** [comment]
> Hi! It was fixed by #933. The fix will be there in the next version.

OK, Thanks!
